.github/workflows/build-and-push-image.yaml

```diff
@@ -23,4 +23,4 @@ jobs:
       -
         name: Run Buildx
         run: |
-          docker build --platform linux/amd64,linux/arm64,linux/arm/v7 -t mirioeggmann/cloudflare-ddns:latest -t mirioeggmann/cloudflare-ddns:$GITHUB_SHA --push . 
+          docker build --platform linux/amd64,linux/arm64,linux/arm/v7 -t ${{ secrets.DOCKER_REPO }}:latest -t ${{ secrets.DOCKER_REPO }}:$GITHUB_SHA --push . 
```

Comment:
One thing, could you change the image tag from :latest to something different like :0.2.0?

Reply:
On the other hand we still have the GITHUB_SHA tag so it should be ok to overwrite the latest tag